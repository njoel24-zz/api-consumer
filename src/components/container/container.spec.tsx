import React from 'react';
import renderer from 'react-test-renderer';
import Container from './container.component';

describe('Container', () => {
	it('renders component and compare with snapshot', () => {
		const component = renderer.create(<Container/>);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});