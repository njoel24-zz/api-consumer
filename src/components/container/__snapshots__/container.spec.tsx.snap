// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders component and compare with snapshot 1`] = `
<div>
  <div>
    <div>
      <div>
        <div />
        <div />
      </div>
      <div>
        <div />
        <div />
      </div>
      <div>
        <div />
        <div />
      </div>
    </div>
    <div>
      <div>
        <div>
          <div />
          <div />
        </div>
        <div>
          <div />
        </div>
        <div>
          <div />
          <div />
        </div>
      </div>
    </div>
  </div>
</div>
`;
